README.rst

```diff
@@ -8,113 +8,115 @@ Masu
 About
 ~~~~~
 
-Data ingestion engine for project-koku responsible for gathering cost and usage.
+Data ingestion engine for project-koku. Masu is responsible for gathering data about cost and resource usage.
+
+The Masu application contains several components - a web service, message bus, and workers. Masu also uses the Koku database. Configuration and management of the database are controlled from the Koku API application.
 
 Getting Started
 ===============
 
-This is a Python project developed using Python 3.6. Make sure you have at least this version installed.
+A basic deployment configuration is contained within the `openshift template files <https://github.com/project-koku/masu/blob/master/openshift>`__. These templates should be acceptable for most use cases.
 
-Development
```

Comment:
Let's leave some of the local development stuff that was there under the development section.  It's important to include both ways. We could sub-section it out into local and openshift dev?